flink-connector-bigquery/pom.xml

```diff
@@ -74,6 +74,12 @@ under the License.
             <groupId>dev.failsafe</groupId>
             <artifactId>failsafe</artifactId>
         </dependency>
+		<!-- joda dependency -->
+		<dependency>
+			<groupId>joda-time</groupId>
+			<artifactId>joda-time</artifactId>
+			<version>2.10.10</version>
```

Comment:
As done in other dependencies, store version as pom property in root pom and use here.

Reply:
Fixed.